test/server_test.py

```diff
@@ -99,7 +99,7 @@ def __init__(self):
         self.unix_socket = os.path.join(self.temp_dir, 'luigid.sock')
 
     def run_server(self):
-        luigi.server.run(unix_socket=unix_socket)
+        luigi.server.run(unix_socket=self.unix_socket)
```

Comment:
@graingert, can you please make sure things are properly tested? This is worrisome. :-/